### PR TITLE
BROKERS: Post Comment

### LINKED ISSUE #52: BROKERS: Post Comment

### PR DESCRIPTION
**Pull request by cjdutoit**
Closes #52

### CODE REVIEW

File: Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+// ---------------------------------------------------------------
+
+using System.Threading.Tasks;
+using Taarafo.Portal.Web.Models.Comments;
+
+namespace Taarafo.Portal.Web.Brokers.API
+{
+    public partial class ApiBroker
+    {
+        private const string commentsRelativeUrl = "api/comments";
```
Comment (hassanhabib) [inline Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs:13]: Constants must start with upper case `CommentsRelativeUrl`

Comment (cjdutoit) [reply Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs]: @hassanhabib fixed

File: Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+using Taarafo.Portal.Web.Models.Comments;
+
+namespace Taarafo.Portal.Web.Brokers.API
+{
+    public partial class ApiBroker
+    {
+        private const string CommentsRelativeUrl = "api/comments";
+
+        public async ValueTask<Comment> PostCommentAsync(Comment comment) =>
+                    await this.PostAsync(CommentsRelativeUrl, comment);
```
Comment (hassanhabib) [inline Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs:16]: @cjdutoit this just needs one tab

Comment (cjdutoit) [reply Taarafo.Portal.Web/Brokers/API/ApiBroker.Comments.cs]: @hassanhabib fixed

**Pull request merged**